Encounter Me/Pages/Index.razor

```diff
@@ -1,17 +1,111 @@
 ﻿@page "/"
 
-@inject IJSRuntime JSRuntime
+@using GoogleMapsComponents
+@using GoogleMapsComponents.Maps
+@using GoogleMapsComponents.Maps.Coordinates
+@using GoogleMapsComponents.Maps.Drawing
 
-<h1>Display Google Map</h1>
-<div id="map" style="height:1000px;width:100%;">
+
+<h1>Google Map</h1>
+<div style="height: 1000px; width: 100%; ">
+    <GoogleMap @ref="@map1" Id="map1" Options="@mapOptions" Height="100%" OnAfterInit="async () => await OnAfterMapInit()"></GoogleMap>
 </div>
+@functions {
+
+    private GoogleMap map1 = default;
```

Comment:
Just a tiny observation -  why the name of variable is "map1" and not just "map"? I though we will have only main map, and yes, that can change in the future. But if it came to pass, wouldn't it be better to name the maps according to their tasks? For example "mainMap", "userMap" et ct.?
P.S I also wanted to try out how commenting works in GitHub.

Reply:
no specific reason, interop package examples followed same naming, therefore I just left it in like that, could change name in the future commits though